### PR TITLE
Mark STLPort support deprecated

### PR DESCRIPTION
**Pull request by var-const**

### CODE REVIEW

File: cmake/external/firestore.cmake [hunk 1]
```diff
@@ -18,9 +18,7 @@ if(TARGET firestore)
   return()
 endif()
 
-# Use a Firestore version slightly after 7.5.1, as it
-# contains a minor crash fix we'd like to include.
-set(version 5e76aa968bfce5e9c8fb0f28f523793f0e9ed5f3)
+set(version CocoaPods-7.7.0)
```
Comment (jonsimantov) [inline cmake/external/firestore.cmake:21]: If we're updating to Cocoapods 7.10.0 today, do I need to update this in my PR?

File: release_build_files/readme.md [hunk 2]
```diff
@@ -571,6 +571,31 @@ code.
 
 ## Release Notes
 
+### 7.3.0
```
Comment (jonsimantov) [inline release_build_files/readme.md:574]: Please put this into 7.2.0, we're putting together that version shortly today.

File: cmake/external/firestore.cmake (hunk 1 above)
Comment (var-const) [reply cmake/external/firestore.cmake:21]: Sorry, I branched this off an older version of `main`. Closing in favor of #366.

File: release_build_files/readme.md (hunk 2 above)
Comment (var-const) [reply release_build_files/readme.md:574]: Done (in #366).

**Pull request closed without merging**